### PR TITLE
OSDOCS-3530: Added links to Google Docs for VPC information.

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
Enterprise-4.11+

Issue:
[OSDOCS-3530](https://issues.redhat.com/browse/OSDOCS-3530)

Link to docs preview:
- [OSD](https://52750--docspreview.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster.html#additional-resources_osd-creating-a-cluster-on-gcp)

Added this reference to the prerequisites:
UpdatedPreReq

Additional information:
Added two links to the Google documentation in the additional resources in the GPC cluster creation topic.

### CODE REVIEW

File: modules/osd-create-cluster-ccs.adoc [hunk 1]
```diff
@@ -199,7 +199,7 @@ endif::osd-on-aws[]
 ifdef::osd-on-gcp[]
 GCP
 endif::osd-on-gcp[]
-VPC, provide your *Virtual Private Cloud (VPC) subnet settings* and select *Next*.
+VPC, provide your *Virtual Private Cloud (VPC) subnet settings* and select *Next*. This process requires the creation of the Cloud network address translation (NAT) as well as a Cloud router. See the additional resources for information on Cloud NATs and Google VPCs.
```
Comment (mburke5678) [inline modules/osd-create-cluster-ccs.adoc:202]: > This process requires

Who is responsible for creating these things?

**Comment by mburke5678** [comment]
@EricPonvelle One small nit. Otherwise LGTM

File: modules/osd-create-cluster-ccs.adoc (hunk 1 above)
Comment (EricPonvelle) [reply modules/osd-create-cluster-ccs.adoc:202]: @mburke5678 it is assumed that whomever is creating this cluster with a VPC would have those resources created, either themselves or someone in their organization.

Comment (mburke5678) [reply modules/osd-create-cluster-ccs.adoc:202]: How about: You must have created the Cloud network address translation (NAT) as well as a Cloud router

Comment (EricPonvelle) [reply modules/osd-create-cluster-ccs.adoc]: @mburke5678 I dig it! Updated the PR

**Pull request merged**